content-src/components/StartupOverlay/_StartupOverlay.scss

```diff
@@ -299,15 +297,15 @@
 
 .error {
   display: none;
-}
 
-html[dir="rtl"] .error{
-  right: 50px;
-  left: auto;
-  
-  &::before {
-    right: 12px;
+  &:dir(rtl) {
```

Comment:
I don't think the rewrite means the same thing:
```
html[dir] .error
```
vs
```
.error:dir(rtl)
```

Reply:
i'll just change the left/right to use offset-inline-start and everything will sort itself out in rtl!